src/app/debug/table/table.component.ts

```diff
@@ -582,4 +589,31 @@ export class TableComponent implements OnInit, OnDestroy {
       this.reportsInProgressThreshold = time;
     });
   }
+
+  handleFilterErrorContext(): string {
+    let result: string = '';
+    let moreThanOne: boolean = false;
+    for (const [key, value] of this.filterErrorDetails) {
+      let typeLabel: string = key;
+      switch (typeLabel) {
+        case 'int': {
+          typeLabel = 'number';
+          break;
+        }
+        case 'long': {
+          typeLabel = 'decimal number';
+          break;
+        }
+        case 'timestamp': {
+          typeLabel = 'date time';
+          break;
+        }
+      }
+      result += moreThanOne
+        ? ", Search value '" + value + "' is not a valid '" + typeLabel + "'"
+        : "Search value '" + value + "' is not a valid '" + typeLabel + "'";
```

Comment:
Can't this be solved better with Template Literals?

Reply:
You're right. I added it in the new commit.